vmware_aria_operations_integration_sdk/mp_init.py

```diff
@@ -319,8 +326,28 @@ def main() -> None:
         #     description="The language for the Management Pack determines the language for the template\n"
         #     "source and build files.",
         # )
-        # create project_directory
+        template_style = selection_prompt(
+            "Select an entry point template for your project",
+            items=[
+                (
+                    TEMPLATE_ADAPTER_ENTRYPOINT_OPTION_KEY,
+                    "Template Adapter Entry Point",
+                ),
+                (
+                    BOILERPLATE_ENTRYPOINT_OPTION_KEY,
+                    "Boilerplate Entrypoint (Advanced users)",
+                ),
+            ],
+            description="- `Template Adapter Entry Point`: Ideal for first-time users; it comes with a template adapter\n"
+            "that collects several objects and metrics from the container the adapter is running. The template\n"
+            "adapter has comments throughout its code that explain what the code does and how to customize\n"
+            "it for your adapter.\n "
+            "- `Boilerplate Entrypoint`: Ideal for experienced users; it comes with the necessary code\n"
+            "and comments to implement test connection, collection, adapter definition, and endpoints logic.\n\n"
+            "For additional information visit https://vmware.github.io/vmware-aria-operations-integration-sdk/get_started/#template-project-and-entry-point",
```

Comment:
I think this is a bit wordy for help text, especially since we're linking to documentation that essentially has the same wording.